<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamLogin Desktop Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="accountsActivate">
      <property name="text">
       <string>Activate Accounts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountsDeactivate">
      <property name="text">
       <string>Deactivate Accounts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="autoStart">
      <property name="text">
       <string>Activate accounts when I start this program</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="autoQuit">
      <property name="text">
       <string>Deactivate accounts when I quit this program</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="systemStartup">
      <property name="text">
       <string>Start program with system startup</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="startMinimized">
      <property name="text">
       <string>Start this program minimized</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="systemTray">
      <property name="text">
       <string>Use an icon in the system tray</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>&amp;App</string>
    </property>
    <addaction name="actionAppExit"/>
   </widget>
   <widget class="QMenu" name="menuAccounts">
    <property name="title">
     <string>A&amp;ccounts</string>
    </property>
    <addaction name="actionAccountsEdit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAboutThisProgram"/>
    <addaction name="actionHelpAboutQt"/>
   </widget>
   <addaction name="menuApp"/>
   <addaction name="menuAccounts"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionAppExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAccountsEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionHelpAboutThisProgram">
   <property name="text">
    <string>&amp;About This Program...</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
